src/main/interface/settings.c

```diff
@@ -1088,6 +1088,12 @@ const clivalue_t valueTable[] = {
     { "osd_log_status_pos",         VAR_UINT16  | MASTER_VALUE, .config.minmax = { 0, OSD_POSCFG_MAX }, PG_OSD_CONFIG, offsetof(osdConfig_t, item_pos[OSD_LOG_STATUS]) },
 #endif
 
+#ifdef USE_OSD_STICK_OVERLAY
+    { "osd_stick_overlay_left",        VAR_UINT16  | MASTER_VALUE, .config.minmax = { 0, OSD_POSCFG_MAX }, PG_OSD_CONFIG, offsetof(osdConfig_t, item_pos[OSD_STICK_OVERLAY_LEFT]) },
```

Comment:
These should be following the naming convention for element positions: `<element>_pos`.

Reply:
Done.